services/refreshToken.js

```diff
@@ -0,0 +1,21 @@
+import authenticationAdapter from 'adapters/Authentication';
+import { clearToken, setToken } from 'helpers/authentication';
+
+export const refreshTokenErrorPath = '/login';
+
+export const refreshToken = async token => {
+  await authenticationAdapter
+    .refreshToken(token)
+    .then(response => {
+      const {
+        data: { attributes },
+      } = response;
```

Comment:
```suggestion
      const attributes = response.data.attributes;
```

JavaScript is quite powerful, but shouldn't we use the good old way here? The same results would fit in 1 line :D 

In case response or data has the risk to be `undefined`, we can use the `.?` operator.

Reply:
done in d001d237faad2946d3f13cd8a0e2207485664da4